docs/troubleshooting.md

```diff
@@ -12,4 +12,28 @@ To connect on our servers you need a working yubikey.
 5) Check available debian package versions: `ll -trh /srv/local-apt-repository`
 6) Check if latest database migration was applied: `sudo -u backend psql entitlementcard` then `SELECT * from migrations;`
 7) Create admin/store manager accounts: `sudo -u backend /opt/ehrenamtskarte/backend/bin/backend create-admin <command>`. You can find example in [runConfigs](../.idea/runConfigurations)
-8) Check if webhook for the debian package installation worked: `journalctl -x --since "48 hours ago" | grep webhook` and find info about the debian package installation: `vim /var/log/apt/history.log`
\ No newline at end of file
+8) Check if webhook for the debian package installation worked: `journalctl -x --since "48 hours ago" | grep webhook` and find info about the debian package installation: `vim /var/log/apt/history.log`
+
+### Import EAK accepting stores
+
+Import of EAK accepting stores on prod and staging envs run on **every odd day** of the month.
```

Comment:
🙃 sorry for the nit-picking i would like to avoid shortcuts as much as possible in documentation. Personal preference
```suggestion
Import of EAK accepting stores on production and staging environments run on **every odd day** of the month.
```

Reply:
Ok, I'll adjust that